docs/standard/linq/toc.yml

```diff
@@ -0,0 +1,346 @@
+- name: LINQ Overview.md
+  href: linq-overview.md
+- name: LINQ to XML
+  items:    
+  - name: Getting Started (LINQ to XML)
+    items:
+    - name: LINQ to XML Overview
+      href: linq-to-xml-overview.md
+    - name: LINQ to XML vs. DOM
+      href: linq-to-xml-vs-dom.md
+    - name: LINQ to XML vs. Other XML Technologies
+      href: linq-to-xml-vs-other-xml-technologies.md
+  - name: Programming Guide (LINQ to XML)
+    items:
+    - name: LINQ to XML Programming Overview
+      items:
+      - name: Functional vs. Procedural Programming (LINQ to XML)
+        href: functional-vs-procedural-programming-linq-to-xml.md
+      - name: LINQ to XML Classes Overview
+        href: linq-to-xml-classes-overview.md
+      - name: XElement Class Overview
+        href: xelement-class-overview.md
+      - name: XAttribute Class Overview
+        href: xattribute-class-overview.md
+      - name: XDocument Class Overview
+        href: xdocument-class-overview.md
+      - name: "How to: Build LINQ to XML Examples"
+        href: how-to-build-linq-to-xml-examples.md
+    - name: XML Trees
+      items:
+      - name: Functional Construction (LINQ to XML)
+        href: functional-construction-linq-to-xml.md
+      - name: Creating XML Trees
+        href: creating-xml-trees.md
+      - name: Parsing XML
+        items:
+        - name: "How to: Parse a String"
+          href: how-to-parse-a-string.md
```

Comment:
@gewarren we have been removing how to from file names, correct?

Reply:
For a new file, it might make sense to leave it off. We aren't supposed to include small words like articles (the, a) either. So maybe just parse-string.md.